packages/python-google-compute-engine/google_compute_engine/distro_lib/helpers.py

```diff
@@ -68,6 +69,23 @@ def CallDhclientIpv6(interfaces, logger, dhclient_script=None,
                      interfaces)
     return
 
+  # Check for a 'tentative' IPv6 address which would prevent `dhclient -6` from
+  # succeeding below. This should only take 1 second, but we try for up to 5.
+  command = ['ip', '-6', '-o', 'a', 's', 'dev'] + interfaces + ['scope', 'link',
```

Comment:
Please line wrap after the open bracket.

```
  command = ['ip', '-6', '-o', 'a', 's', 'dev'] + interfaces + [
      'scope', 'link', 'tentative']
```

Reply:
Will it work to run this command on multiple interfaces at the same time?

Reply:
no, but the fact that this entire rigging uses the name 'interfaces' is misleading to begin with. these functions only pass around a list containing the default interface.
can add a for loop around the check here, if we want to extend the fiction that this feature supports multiple interfaces.

Reply:
relevant code in NetworkDaemon.HandleNetworkInterfaces():
```
     if self.network_setup_enabled:                                              
       default_interface = network_interfaces[0]                                 
       if default_interface.ipv6:                                                
          self.network_setup.EnableIpv6([default_interface.name])                 
       else:                                                                     
         self.network_setup.DisableIpv6([default_interface.name])                
       self.network_setup.EnableNetworkInterfaces(                               
           [interface.name for interface in network_interfaces[1:]])  
```

Reply:
Interesting. I do thing that we should either ensure there is only one entry or loop over the list. Either is fine, but we shouldn't use the list in a way that would break us if the list ever contained more than one entry.